pkg/handlers/app/app.go

```diff
@@ -17,7 +17,9 @@ limitations under the License.
 package app
 
 import (
+	"fmt"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"strings"
```

Comment:
imports should be sorted 😛 
```go
import (
	"fmt"
	"strings"

	"k8s.io/apimachinery/pkg/runtime/schema"
)

```

Reply:
Damn, I don't know why the editor didn't do it automatically 😞